deploy.sh

```diff
@@ -69,7 +72,7 @@ fi
 
 deploy/bin/import-docker-worker-secrets
 trap 'rm -rf /tmp/docker-worker*' EXIT
-deploy/bin/build app
+deploy/bin/build $BUILD_TARGET
 if [ "$DEPLOYMENT" == "taskcluster-net" ]; then
     deploy/bin/update-worker-types.js $*
```

Comment:
This only makes sense for `app`, doesn't it?

Reply:
Good point!